src/Controls/src/Core/BindableObject.cs

```diff
@@ -783,11 +786,11 @@ internal class BindablePropertyContext
 		{
 			public BindableContextAttributes Attributes;
 
-			public SortedList<SetterSpecificity, BindingBase> Bindings = new();
+			public SetterSpecificityList<BindingBase> Bindings = new();
 
 			public Queue<SetValueArgs> DelayedSetters;
 			public BindableProperty Property;
-			public readonly SetterSpecificityList Values = new();
+			public readonly SetterSpecificityList<object> Values = new(3);
```

Comment:
When the context is created, we always set the `DefaultValue` specificity in there, so it makes no sense to initialize the collection as an empty one to resize it on the next statement.